packages/shared/src/components/modals/squads/SubmitSquadForReviewModal.tsx

```diff
@@ -79,14 +87,13 @@ const SubmitSquadForReviewModal = ({
         <Checklist>
           <ChecklistItem iconTag={MegaphoneIcon}>
             The posts I shared meet the{' '}
-            <a
-              className="text-text-link underline hover:no-underline"
+            <Anchor
               href="https://docs.daily.dev/docs/for-content-creators/content-guidelines"
```

Comment:
I know this is old, but don't we have a rebrandly here?
Can we add it to backlog?

Reply:
Found one at `constants` file. Let me update this now.